_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -1,19 +1,17 @@
-<!-- A general description of the requirement can be given here. -->
+This guide will explain the end-to-end flow of sharing and answering a [Request]({% link _docs_integrate/data-model-overview.md %}#request) via a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
+Usually, this flow happens between a [Connector]({% link _docs_explore/52-connector.md %}) and the [App]({% link _docs_explore/50-app.md %}), but for simplicity and more transparency we will use two Connectors in this example.
+Thus, you'll need two Connectors, that either already have a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with each other or not yet.
 
-This guide will explain the end to end flow of sharing and answering a [Request]({% link _docs_integrate/data-model-overview.md %}#request) over a Template. This flow usually happens between the App and a Connector, but for simplicity and more transparency we will use two Connectors. Therefore you have to start two Connectors that don't have a Relationship yet.
+You can use the [Connector Installation Guide]({% link _docs_operate/setup-with-docker-compose.md %}) if you need help setting up the Connectors.
 
-You can use the [Connector Installation Guide]({% link _docs_operate/setup-with-docker-compose.md %}) if you need help for the setup the Connectors.
-
-<!-- This include inserts the table with the metadata  -->
-
-{% include properties_list.html %}
-
-On the first Connector you will create a Template. This Connector will be called Templator in the following steps. The second Connector is called Requestor, because it will create the Relationship and therefore it creates the `RelationshipCreationChangeRequest`.
+On the first Connector, which we will refer to as the Sender, you will create the Request and the RelationshipTemplate.
+The second Connector, which we will refer to as Recipient, will receive the RelationshipTemplate and respond to the Request.
 
 ## Check your Request's validity
 
-At first you should check if your Request is valid. You can do this by calling the `POST /api/v2/Requests/Outgoing/Validate` route on the Templator Connector with the following body.
-For simplicity the Request inside the Template only contains an AuthenticationRequestItem, but you can use any [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitems) you want.
+Firstly, you should [check if your Request is valid]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}).
+As an example, we use a Request with just an [AuthenticationRequestItem]({% link _docs_integrate/data-model-overview.md %}#authenticationrequestitem), but you can use any Request you want.
+For an overview of the available [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitems), checkout our [Request and Response Introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
```

Comment:
We should write "Request and Response **i**ntroduction" here or change the title of the corresponding scenario to "Request and Response Introduction". I'd prefer the first option.